pkg/products/rhsso/prometheusRules.go

```diff
@@ -71,6 +72,77 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 					},
 				},
 			},
+
+			//SLO-availability-alerts:
+			//https://sre.google/workbook/alerting-on-slos/
+			//https://promtools.dev/alerts/errors
+			{
+				AlertName: "rhsso-slo-availability-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "rhsso-slo-availability.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: fmt.Sprintf("%sRhssoAvailability5mto1hErrorBudgetBurn", strings.ToUpper(installationName)),
+						Annotations: map[string]string{
+							"sop_url": resources.SopUrlSloRhssoAvailabilityAlert,
+							"message": "High 5m and 1h error budget burn for RHSSO",
+						},
+						Expr: intstr.FromString(fmt.Sprintf(`
+							sum( sum(rate(haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace='%s', code="5xx"}[5m]))
```

Comment:
we may not need the regex here. This was to cover that in MAS SSO there are to keycloak routes in staging

Reply:
same for the others below

Reply:
I'm checking Prometheus in RHOAM cluster,
using query: haproxy_backend_http_responses_total {route=~"^keycloak.*", code="5xx"}
There are two keycloak routes: route="keycloak-edge" and route="keycloak"
So seems  current code is correct (?) : route=~"^keycloak.*"

Reply:
There are only two routes for haproxy_backend_http_responses_total 5xx metrics at RHOAM: 
route="keycloak" and route="keycloak-edge"
Could I remove "route=~"^keycloak.*" expession from alert? So remains following:
'haproxy_backend_http_responses_total{exported_namespace=~"redhat-.*sso$"}'
Is it ok?